lib/components/app/popup.tsx

```diff
@@ -0,0 +1,54 @@
+import { Modal } from 'react-bootstrap'
+import { useIntl } from 'react-intl'
+import coreUtils from '@opentripplanner/core-utils'
+import React, { useEffect } from 'react'
+
+type Props = {
+  content: { appendLocale?: boolean; id?: string; url?: string }
+  hideModal: () => void
+}
+
+/**
+ * This component renders a bootstrap modal featuring a URL passed in via a content object.
+ * On mobile, the link is opened in a popup window.
+ *
+ * The modal is automatically shown once a valid URL is passed. A callback is provided for when
+ * the modal is closed by clicking on the background.
+ */
+const PopupWrapper = ({ content, hideModal }: Props): JSX.Element | null => {
+  const intl = useIntl()
+  const { appendLocale, id, url } = content
+  const shown = !!url
+
+  const isMobile = coreUtils.ui.isMobile()
+  // appendLocale is true by default, so undefined is true
+  const compiledUrl = `${url}${
+    appendLocale !== false ? intl.defaultLocale : ''
+  }`
+
+  useEffect(() => {
+    if (isMobile) {
+      window.open(compiledUrl)
```

Comment:
This does not work on Chrome/ium (either mobile view from devTools or on my phone). I suspect that's because these browsers block popups by default. An alternate approach for the mobile view is to pass an `href` and `target="_blank"` props to the buttons (make them react-bootstrap), or directly render `<a>` with these props and desired styling, see https://react-bootstrap-v3.netlify.app/components/buttons/#buttons-tags.

Reply:
I've investigated this and the popups aren't being blocked (as they're being initiated by user action). What is going wrong is the `isMobile` check, which always returns false even on real mobile devices. I'm going to try to find a different mobile check that's more reliable.

Reply:
Update: the issue wasn't the mobile check, but rather that `Popup` wasn't being rendered in the mobile view at all. Now that it is the regular `window.open` works, like it does in other parts of otp-rr